MediaBrowser.Providers/Manager/ItemImageProvider.cs

```diff
@@ -470,7 +470,7 @@ private bool UpdateMultiImages(BaseItem item, IReadOnlyList<LocalImageInfo> imag
             CancellationToken cancellationToken)
         {
             var eligibleImages = images
-                .Where(i => i.Type == type && !(i.Width.HasValue && i.Width.Value < minWidth))
+                .Where(i => i.Type == type && (!i.Width.HasValue || i.Width.Value >= minWidth))
```

Comment:
Comparisons with null return false.
```suggestion
                .Where(i => i.Type == type && i.Width.Value >= minWidth)
```